app/entities/program-metadata/model/useProgramCanonicalMetadata.tsx

```diff
@@ -17,7 +17,7 @@ export function useProgramCanonicalMetadata(
     useSuspense = false
 ) {
     const { data } = useSWRImmutable(
-        `program-metadata-${seed}-${programAddress}-${url}`,
+        `program-metadata-${programAddress}-${url}-${seed}`,
```

Comment:
What does this change?

Reply:
Logic-wise, it does not affect Explorer, but I put the chunk that changes more frequently to the end.